tests/zfs-tests/tests/functional/trim/autotrim_trim_integrity.ksh

```diff
@@ -61,7 +61,7 @@ log_must set_tunable64 TRIM_EXTENT_BYTES_MIN 4096
 typeset trim_txg_batch=$(get_tunable TRIM_TXG_BATCH)
 log_must set_tunable64 TRIM_TXG_BATCH 8
 
-for type in "" "mirror" "raidz" "raidz2" "raidz3"; do
+for type in "" "mirror" "raidz" "raidz2" "draid" "draid2"; do
```

Comment:
Might be due to runtime limits on the test scripts.

Limiting these to "one level per algorithm", is quite consistent with how the same is changed for compression last year. Codepath wise, I doubt raidz2 and raidz3 have any relevant differences when it comes to trim.

Reply:
@Ornias1993 is correct, it was purely to speed up the test case.  Originally I was testing raidz[1-3] and draid[1-3] but found that could take longer than the 10min per-test time limit in the CI causing bogus failures.  To avoid that I drop raidz3.